src/Explorer/Explorer.ts

```diff
@@ -1734,10 +1735,17 @@ export default class Explorer {
         notebookWorkspace.properties.status &&
         notebookWorkspace.properties.status.toLowerCase() === "stopped"
       ) {
+        notificationId = NotificationConsoleUtils.logConsoleMessage(

```

Comment:
There is a new set of methods you can use for this that is less verbose:

`const clearMessage = NotificationConsoleUtils.logConsoleProgress("message")`

... later

`clearMessage()`